health/health.c

```diff
@@ -371,6 +372,10 @@ static inline void health_alarm_log_process(RRDHOST *host) {
         ALARM_ENTRY *t = ae->next;
 
         health_alarm_log_free_one_nochecks_nounlink(ae);
+        if(likely(!alarm_entry_isrepeating(host, ae))) {
+            health_alarm_log_free_one_nochecks_nounlink(ae);
```

Comment:
Could this be a double free situation here?

Reply:
Good catch, Thiago added the fix in PR #6379